src/app/lib/valinta-tulos-service.ts

```diff
@@ -489,6 +489,27 @@ export const sendVastaanottopostiHakemukselle = async (
   return response.data as string[];
 };
 
+export const sendVastaanottopostiHakukohteelle = async (
+  hakukohdeOid: string,
+): Promise<string[]> => {
+  const response = await client.post(
+    `${configuration.vastaanottopostiHakukohteelleUrl({ hakukohdeOid })}`,
+    { hakukohdeOid },
+  );
+  return response.data as string[];
+};
+
+export const sendVastaanottopostiValintatapaJonolle = async (
+  hakukohdeOid: string,
+  valintatapajonoOid: string,
+): Promise<string[]> => {
+  const response = await client.post(
```

Comment:
Response-tyyppi pitäs pystyä antaamaan post-funkkarillekin, eli ei pitäs tarvita tehdä `as string[]`
```suggestion
  const response = await client.post<Array<string>>(
```